src/gui/guiFormSpecMenu.cpp

```diff
@@ -2686,7 +2714,7 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 					core::rect<s32>(size.X/2-70, pos.Y,
 							(size.X/2-70)+140, pos.Y + (m_btn_height*2));
 			const wchar_t *text = wgettext("Proceed");
-			Environment->addButton(mydata.rect, this, 257, text);
+            GUIButton::addButton(Environment, mydata.rect, this, 257, text);
```

Comment:
Erroneous indent.

Reply:
Fixed, thanks for the catch!
My text editor is still set up with spaces instead of tabs, so I had to do a cleanup sweep after writing the code...